src/javascript/_common/base/elevio.js

```diff
@@ -1,18 +1,20 @@
-const ClientBase    = require('./client_base');
-const GTM           = require('./gtm');
-const BinarySocket  = require('./socket_base');
-const getLanguage   = require('../language').get;
-const localize      = require('../localize').localize;
-const createElement = require('../utility').createElement;
+const ClientBase      = require('./client_base');
+const GTM             = require('./gtm');
+const BinarySocket    = require('./socket_base');
+const languageFromUrl = require('../language').urlLang;
+const localize        = require('../localize').localize;
+const createElement   = require('../utility').createElement;
 
 const Elevio = (() => {
     const init = () => {
         if (!window._elev) return; // eslint-disable-line no-underscore-dangle
         window._elev.on('load', (elev) => { // eslint-disable-line no-underscore-dangle
-            const available_elev_languages = ['es', 'id', 'pt', 'ru'];
-            const current_language         = getLanguage().toLowerCase();
+            const available_elev_languages = ['de', 'fr', 'id', 'it', 'po', 'pt', 'ru', 'vi'];
+            const current_language         = languageFromUrl().toLowerCase();
```

Comment:
getLanguage also calls languageFromUrl, what was the issue that this is solving?

Reply:
noted and addressed.